platform-emf/src/main/resources/com/softicar/platform/emf/emf-form-style.css

```diff
@@ -78,9 +95,7 @@
 	grid-column: 1/3;
 }
 
-.EmfFormCommonActionsDiv {
-	gap: 4px;
-	display: flex;
+.DomActionBar.EmfFormCommonActionsDiv {
```

Comment:
Do handcrafted context menus still look like before now?

Reply:
Well, EmfFormCommonActionsDiv extends DomActionBar, so nothing should have changed
If you mean handcrafted context menus like in the module classes page, then yes, everything is the same

Reply:
Ooops, misread the class name. Never mind.